### ISSUE TITLE
What CCSID should the system value have and then what value should the user ID have?

### ISSUE THREAD

**Issue by a4ginatl** [codefori/vscode-ibmi#2209]
When I start VSC, I get the following message.

The system QCCSID is not set correctly. We recommend changing the CCSID on your user profile first, and then changing your system QCCSID. Runtime CCSID detected as -2 and is invalid. Please change the CCSID or default CCSID in your user profile. Using fallback methods to access the IBM i file systems.

My system value is set to use 37.
My user ID is set to 37.

 The system does sign on and work.

What does VSC expect of the CCSIDs?

TIA, Darryl Freinkel.

<hr />

⚠️ **REMOVE THIS LINE AND ANY SENSITIVE INFORMATION BELOW!** ⚠️

|Context|Version|
|-|-|
|Code for IBM i version|2.12.1|
|Visual Studio Code version|1.92.0|
|Operating System|win32_x64|

<details>
<summary>Active extensions</summary>

```
COBOL (cobol): 10.7.23
Code Coverage for IBM i (code-coverage-ibmi): 0.1.7
Code for IBM i Walkthroughs (vscode-ibmi-walkthroughs): 0.5.0
Db2 for IBM i (vscode-db2i): 1.1.1
Git (git): 1.0.0
Git Base (git-base): 1.0.0
GitHub (github): 0.0.1
IBM i Notebooks (vscode-ibmi-notebooks): 0.0.5
Merge Conflict (merge-conflict): 1.0.0
Node Debug Auto-attach (debug-auto-launch): 1.0.0
TODO Highlight (vscode-todo-highlight): 1.0.5
```
</details>

<hr />

**_Not connected_** 🔌

**Comment by worksofliam** [comment]
@a4ginatl what happens if you right click on your connection (before connecting) and try the "connect and reload" option - does that solve it?

**Comment by a4ginatl** [comment]
I did that and it made no difference.
FYI, this is a v5.4 system that I have used VSC on for a while, but not much. 
Maybe a parameter error based on the OS version?
Thanks, Darryl.

**Comment by a4ginatl** [comment]
I guess something changed in one of the updates. It now does not expand the filters. 
It is also not loading the library list.
Is there a log to look at?
I have granted authority to the user ID again. The system is quite locked down.

Darryl

**Comment by worksofliam** [comment]
@a4ginatl FYI, we don't support anything below 7.3.

Please share the contents of your Code for IBM i output channel after the list doesn't appear.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/3708366/178739532-03c8ad30-84de-4698-9904-65938f2ef651.png">

**Comment by a4ginatl** [comment]
I am not able to get that screen loaded to be able to view the OUTPUT tab.

I get this message when expanding a filter.

Token EPOCH was not valid. Valid tokens: DAY HOUR YEAR MONTH MINUTE SECOND. (42601)

**Comment by worksofliam** [comment]
@a4ginatl Yes, that's only available on 7.3+. If you're on 5.4, there is not much we can do about that.
